### DISCUSSION TITLE
Why function completion doesn't fill brackets automatically?

### DISCUSSION THREAD

**Discussion by winkee01** [hrsh7th/nvim-cmp#944] [Q&A]
For example, in the below case, when I type `fmt.Println`, 

![image](https://user-images.githubusercontent.com/49930785/165700202-02e4da87-0019-4c2c-baca-bd03e64c3981.png)

and then hit `Enter` to confirm the completion selection, it becomes

![image](https://user-images.githubusercontent.com/49930785/165700405-73e4765e-ee86-400c-9c98-caa34f5fbc50.png)

However, this is not what I expected, I expected it to also fill `()` brackets for me, but it didn't, **why?**
Now I have to manually type `()`, which is very unpleasant.

<img width="822" alt="CleanShot 2022-04-28 at 15 32 04@2x" src="https://user-images.githubusercontent.com/49930785/165700775-7fcbf86c-43e4-4bea-a15b-2d6b335df907.png">

**Comment by hrsh7th** [comment]
nvim-cmp is just behaves as Language Server's instruction.
You should re-check your gopls server configuration.

I think you sgould set it up 'usePlaceholders = true'

**Reply by bew** [reply]
Is it possible to force the completion engine to not follow what the server says?

**Reply by winkee01** [reply]
I tried 'usePlaceholders = true', it doesn't work.
